### PR TITLE
FCREPO-1797. Added scripts to test backups and restore of fcrepo home

### PR DESCRIPTION
**Pull request by mohideen**

### CODE REVIEW

**Comment by mohideen** [comment]
https://jira.duraspace.org/browse/FCREPO-1797

**Comment by awoods** [comment]
@ruebot, I am waiting for @mohideen to put the associated ticket into "review" before reviewing this potentially in-flight work.

File: README.md [hunk 1]
```diff
@@ -50,6 +50,29 @@ You can shell into the machine with `vagrant ssh` or `ssh -p 2222 vagrant@localh
...
 * `vagrant up`
 
+#### Using the backup and restore scripts
+The scripts at the ~/backup_restore directory can be used to test backing up and restoring the fedora repository for consistency.
+
+The following command will cause 50 parallel processes to load data to the repository while creating snapshots of fcrepo home directory every 2 seconds.
+
+```
+cd ~/backup_restore/
+./hot_backup_runner.sh 50 /var/lib/tomcat7/fcrepo4-data 2
```
Comment (awoods) [inline README.md:60]: When I run this command, after a few minutes I run out of disk space with:
`cp: cannot create directory ‘./backups/fcrepo_home.bkp.20151213181231’: No space left on device`
How much disk is needed to run this? Is there a way of specifying or limited the required disk space?

Comment (mohideen) [inline README.md:60]: @awoods, the amount of space required depends on the input params. For example, `./hot_backup_runner.sh 50 /var/lib/tomcat7/fcrepo4-data 2` will run `50` parallel ingest processes and create backups every `2` seconds. You can reduce the backup frequency to 5 or 10 seconds, if you are running out of storage.

**Comment by awoods** [comment]
Resolved with: https://github.com/fcrepo4-exts/fcrepo4-vagrant/commit/f359465359634b5bddff1fc98db48b096f4e6de3

**Pull request closed without merging**